.rubocop.yml

```diff
@@ -7,4 +7,10 @@ Metrics/LineLength:
   # Based on https://github.com/PyCQA/flake8-bugbear#opinionated-warnings (`B950`)
   Max: 88
 
+# libvirt-formula overredies
+Lint/AmbiguousRegexpLiteral:
+  # Exclude { should match /<REGEX>/ }
+  Exclude:
+    - 'test/integration/default/controls/config_spec.rb'
+
```

Comment:
This whole exclusion section won't  be required with the two fixes I've proposed.